### PR TITLE
3rd party tools

### PR DESCRIPTION
**Pull request by ojemuyiwa**
Configured hotjar for production and sonarcloud

#TISNEW-3933

### CODE REVIEW

File: projects/hee-shared/src/lib/hotjar/hotjar.module.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { NgModule, InjectionToken, Inject, Optional, SkipSelf, ModuleWithProviders } from '@angular/core';
+import { InjectScript } from '../utilities.functions';
+import { DOCUMENT } from '@angular/common';
+
+export interface HotjarConfig {
+  hotjarid: number;
```
Comment (omarCreativeDev) [inline projects/hee-shared/src/lib/hotjar/hotjar.module.ts]: is it possible to camelcase this interface?

File: projects/hee-shared/src/lib/hotjar/hotjar.module.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  enabled: boolean;
+}
+
+const HotjarConfigValue = new InjectionToken<HotjarConfig>('HotjarConfig');
+
+@NgModule({
+  declarations: []
+})
+export class HotjarModule {
+  private hotjarscript: HTMLScriptElement;
```
Comment (omarCreativeDev) [inline projects/hee-shared/src/lib/hotjar/hotjar.module.ts]: is it possible to camelcase this property?

File: projects/hee-shared/src/lib/hotjar/hotjar.module.ts (hunk 1 above)
Comment (ojemuyiwa) [reply projects/hee-shared/src/lib/hotjar/hotjar.module.ts]: Revised all and camel cased across using angular style guide: https://angular.io/guide/styleguide

**Comment by omarCreativeDev** [comment]
pr looks good but seems like prettifier hasn't run on your commits. are you using source tree to commit and push? if so not sure how husky would come in to play to auto prettify modified files and also run js and css linting. maybe there is a way to configure source tree to use husky or auto prettify and lint files?

or alternatively if you commit via the command line everything will be automagically done for you as husky pre-commit hook will prettify and run linting before allowing to commit

**Pull request merged**